de.bund.bfr.knime.fsklab.nodes/src/de/bund/bfr/knime/fsklab/v1_9/fskdbview/FSKDBViewRepresentation.java

```diff
@@ -0,0 +1,111 @@
+/*
+ ***************************************************************************************************
+ * Copyright (c) 2020 Federal Institute for Risk Assessment (BfR), Germany
+ *
+ * This program is free software: you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program. If
+ * not, see <http://www.gnu.org/licenses/>.
+ *
+ * Contributors: Department Biological Safety - BfR
+ *************************************************************************************************
+ */
+package de.bund.bfr.knime.fsklab.v1_9.fskdbview;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.knime.core.node.InvalidSettingsException;
+import org.knime.core.node.NodeSettingsRO;
+import org.knime.core.node.NodeSettingsWO;
+import org.knime.js.core.JSONDataTable;
+import org.knime.js.core.JSONViewContent;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+
+@JsonAutoDetect
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, include = JsonTypeInfo.As.PROPERTY, property = "@class")
+public class FSKDBViewRepresentation extends JSONViewContent {
+
+  private static final String TABLE_ID = "tableid";
+
+  private String tableID;
+
+  public String getTableID() {
+    return tableID;
+  }
+
+  public void setTableID(String tableID) {
+    this.tableID = tableID;
+  }
+
+  private JSONDataTable m_table;
```

Comment:
Regarding the naming it is fine it we follow KNIME naming preceding class variables of KNIME classes with m_ but then we should be consistent and rename tableID.